pandas/tseries/tests/test_resample.py

```diff
@@ -1118,6 +1118,43 @@ def test_resample_loffset_count(self):
 
         assert_series_equal(result, expected)
 
+    def test_resample_loffset_agg(self):
+        # GH 13218
+        # test agg() with timeseries and df w/ DateTimeIndex
+        rng = date_range('1/1/2000', periods=50, freq='S')
+        vals = np.random.randn(50)
+        expected_means = [vals[:10].mean(), vals[10:20].mean(),
+                          vals[20:30].mean(), vals[30:40].mean(),
+                          vals[40:50].mean()]
+        expected_index = (date_range('1/1/2000', periods=5, freq='10S') +
+                          timedelta(hours=2))
+        ts = Series(vals, index=rng)
+
+        result = ts.resample('10s', loffset='2h').agg('mean')
+        expected = Series(expected_means, index=expected_index)
+
+        assert_series_equal(expected, result)
+
+        df = DataFrame({'A': vals, 'B': vals}, index=rng)
+
+        result = df.resample('10S', loffset='2h').agg({'A': 'count',
+                                                       'B': 'mean'})
+        expected = DataFrame({'A': 10, 'B': expected_means},
+                             index=expected_index)
+        assert_frame_equal(result, expected)
+
+        # now test agg() with df w/ PeriodIndex
```

Comment:
e.g. this needs to go into the PeriodIndex Test class class. Need a test for TD as well..